packages/web/src/scss/settings/_transitions.scss

```diff
@@ -0,0 +1,12 @@
+@use '@tokens' as tokens;
+
+$duration-fast: 0.125s;
+$duration-medium: 0.25s;
+
+$timing-eased-in-out: ease-in-out;
+$timing-eased-in-out-fast: cubic-bezier(0.4, 0, 0.2, 1);
+
+$scale-ratio-small-objects: 0.95;
```

Comment:
we don't use these? Do we need them?

Reply:
Oh, it's meant for `Tooltip`. I made changes to transitions I didn't push yet as I didn't want to add any more bloat to this PR. I'll clean it up here for now, thanks 👍🏻 .